### ISSUE TITLE
restarting integrated console doesn't use VS Code PS profile

### ISSUE THREAD

**Issue by jeernest** [PowerShell/vscode-powershell#825]
### System Details

- Operating system name and version:
- VS Code version:
- PowerShell extension version:
- Output from `$PSVersionTable`:

```
PS G:\> code -v

1.12.2
19222cdc84ce72202478ba1cec5cb557b71163de
PS G:\> $pseditor.EditorServicesVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
1      1      0      0

PS G:\> code --list-extensions --show-versions

idleberg.hopscotch@0.1.4
ms-vscode.csharp@1.10.0
ms-vscode.PowerShell@1.1.0
qub.qub-msbuild@0.6.3
PS G:\> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.14409.1005
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14409.1005
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Issue Description

I am experiencing a problem with Integrated Console. When I start the Integrated console, it looks like this
```powershell

Starting PowerShell...

PS G:\> $profile

C:\Users\jan.elias\Documents\WindowsPowerShell\Microsoft.VSCode_profile.ps1
PS G:\> _
```
when I for some reason kill the terminal and start it again, it seems like it starts my default console profile:
```powershell

Hello jan.elias welcome on .....

G:\products\sam [feature/swbuild ≡ +0 ~1 -0 | +4 ~7 -0 !]> $profile
C:\Users\jan.elias\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1
G:\products\sam [feature/swbuild ≡ +0 ~1 -0 | +4 ~7 -0 !]> _
```
### Attached Logs

github doesn't allow dragging my .zip file, I dont know why, let me know if you need them

**Comment by daviwil** [comment]
Hey @jeernest, we released an update on Wednesday (1.2.0) which fixes this issue, please check again after installing this update and let us know if it works for you now!

**Comment by SydneyhSmith** [comment]
Closing this issue as inactive, will re-open if this is still an issue.
